elastic/security/templates/component/track-shared-logsdb-mode.json

```diff
@@ -5,15 +5,20 @@
         "index": {
             "mode": {{ index_mode | tojson }},
             {% if synthetic_source_keep and synthetic_source_keep != 'none' %}
-            "mapping": {
-                "synthetic_source_keep": "{{ synthetic_source_keep }}"
-            },
+            "mapping.synthetic_source_keep": {{ synthetic_source_keep | tojson}},
             {% endif %}
             "sort.field": [ "host.id", "@timestamp" ],
             "sort.order": [ "asc", "desc" ],
             "sort.missing": ["_first", "_last"]
         }
         {% endif %}
+      },
+      "mappings": {
+        {% if source_mode %}
+        "_source": {
+          "mapping.source.mode": {{ source_mode | tojson }}
```

Comment:
I don't understand this? Did we introduce something I am not aware of?

Reply:
This was a silly mistake on my side.